src/main/java/org/folio/event/service/UserTenantService.java

```diff
@@ -51,4 +65,15 @@ public Future<Boolean> deleteUserTenant(UserTenant userTenant, String tenantId,
     PostgresClient pgClient = pgClientFactory.apply(vertx, tenantId);
     return pgClient.withConn(conn -> tenantRepository.deleteUserTenant(conn, userTenant, tenantId));
   }
+
+  public Future<Void> isUsernameUnique(Conn conn, String username, String consortiaCentralTenantId) {
+    return tenantRepository.isUserNameAlreadyExists(conn, username, consortiaCentralTenantId)
```

Comment:
lets use the same naming convention: isUsernameUnique or isUserNameAlreadyExists in all places.

Reply:
Renamed to isUsernameAlreadyExists